Correcting quay login command and updating example as per GCP Artifac… (#18044)
yugabyte/yugabyte-db: docs/content/preview/yugabyte-platform/install-yugabyte-platform/prepare-environment/kubernetes.md

  docker pull nginxinc/nginx-unprivileged:1.23.3
  ```

- Login to your target container registry, as per the following example that uses Google Artifact Registry:

>    **Note**: The "Service Account" and "Location" used in the below example have to be replaced as applicable.

Reviewer: ```suggestion
   **Note**: Replace the Service Account and Location in the example as applicable.
```

  ```sh
  gcloud auth activate-service-account yugabytedb-test@yugabytedb-test-384308.iam.gserviceaccount.com --key-file=key.json
  gcloud auth configure-docker us-central1-docker.pkg.dev
  
  ```

> - Tag the local images to your target registry, as follows (Here format for GCP Artifact Registry ):

Reviewer: ```suggestion
- Tag the local images to your target registry. The following example uses Google Artifact Registry.
```

>   **Note**: The Location, Project Id, Repository, and Image used in the below example have to be replaced as applicable.

Reviewer: ```suggestion
  **Note**: Replace the Location, Project ID, Repository, and Image in the example as applicable.
```

  ```sh

>   docker tag quay.io/yugabyte/yugaware:{{<yb-version version="preview" format="build">}} us-central1-docker.pkg.dev/yugabytedb-test-384308/yugabytepoc/yugabyte/yugaware:{{<yb-version version="preview" format="build">}}

Reviewer 2: @rajeev-asthana : can you clarify why we want to change these links from gcr.io to us-central1.docker.pkg.dev?

Author: @iSignal :   GCR  i.e. Google container registry is deprecated now and is replaced by Google Artifact Registry. Example here is using GCR which needs to be updated with Artifact registry example. "us-central1.docker.pkg.dev" is the syntax for pushing images to Artifact registry in GCP.

  docker tag nginxinc/nginx-unprivileged:1.23.3 us-central1-docker.pkg.dev/yugabytedb-test-384308/yugabytepoc/nginxinc/nginx-unprivileged:1.23.3
  docker tag postgres:14.4 us-central1-docker.pkg.dev/yugabytedb-test-384308/yugabytepoc/postgres:14.4
  docker tag tianon/postgres-upgrade:11-to-14 us-central1-docker.pkg.dev/yugabytedb-test-384308/yugabytepoc/tianon/postgres-upgrade:11-to-14
  docker tag prom/prometheus:v2.41.0 us-central1-docker.pkg.dev/yugabytedb-test-384308/yugabytepoc/prom/prometheus:v2.41.0
  ```

- Push images to the private container registry, as follows:

>   **Note**: The Location, Project Id, Repository, and Image used in the below example have to be replaced as applicable.

Reviewer: ```suggestion
  **Note**: Replace the Location, Project ID, Repository, and Image in the example as applicable.
```
